molecule/default/molecule.yml

```
@@ -0,0 +1,52 @@
+---
+dependency:
+  name: galaxy
+  enabled: false
+
+driver:
+  name: containers
+
+platforms:
+  - name: opencast-postgresql-centos9
+    image: quay.io/centos/centos:stream9
+    pre_build_image: false
+    command: /sbin/init
+    privileged: true
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:rw
+      - /run
+      - /run/lock
+    cgroupns_mode: host
```

I would suggest not using privileged mode.

```suggestion
    tmpfs:
      "/tmp": "exec"
      "/run": "rw,noexec,nosuid,nodev"
    volumes:
      - /sys/fs/cgroup:/sys/fs/cgroup:ro
```

I've tested it successfully on my Fedora with podman. Can you test it on your system as well?

---

It is not working on my systems. Without using privileged mode, it shows 'UNREACHABLE! "Failed to create temporary directory.' To solve it, I use privileged mode from the [link](https://github.com/ansible/molecule/issues/3818/)

---

Now it is working by changing from Docker to Podman